gateways/token_balances_api_gateway.py

```diff
@@ -0,0 +1,103 @@
+from typing import List, Optional
+
+from elasticsearch import Elasticsearch
+
+from common.configuration import ELASTIC_RESULTS_PER_PAGE, ELASTIC_SEARCH_TIMEOUT, ELASTIC_TOKEN_BALANCES_INDEX
+from gateways.clients.elastic_search_client import ElasticSearchClient
+from utils.elastic_search.elastic_search_utils import ElasticSearchUtils
+
+SORTABLE_FIELDS = {
+    'address': 'keyword',
+    'token_id': 'keyword',
+    'unlocked_balance': 'long',
+    'locked_balance': 'long',
+    'total': 'long',
+}
+
+
+class TokenBalancesApiGateway:
+    """ Gateway to interact with the Hathor ElasticSearch cluster with the token balances index
+    """
+
+    def __init__(
+        self,
+        elastic_search_client: Optional[Elasticsearch] = None,
+    ) -> None:
+        self.elastic_search_client = ElasticSearchClient(elastic_index=ELASTIC_TOKEN_BALANCES_INDEX,
+                                                         client=elastic_search_client)
+
+    def get_token_information(self, token_id: str) -> dict:
+        """Retrieve total number of addresses and transactions for a given token_id
+        """
+
+        body = {
```

Comment:
index must be informed

Reply:
Fixed in 0b2b67d176977d71660743283b3f0e4f3fe7af1b